lbrynet/daemon/Daemon.py

```diff
@@ -385,7 +386,15 @@ def _clean_up_temp_files(self):
             except OSError:
                 pass
 
+    @staticmethod
+    def _already_shutting_down(sig_num, frame):
+        log.info("Already shutting down")
+
     def _shutdown(self):
+        # ignore INT/TERM signals once shutdown has started
```

Comment:
If you press Ctrl-C while daemon is shutting down, you'll get an error message where it tries to shut down the reactor twice. This prevents that and lets you know the daemon's working on it.